apps/frontend/src/app/institutions/components/georeferencia/georref-map.scss

```
@@ -0,0 +1,4 @@
+.map {
+    width: 100%;
```

```suggestion
    width: 95vw;
```
Es mejor no usar anchos porcentuales para componentes externos que hacen muchos cálculos antes de renderizarse. 
Con eso debería quedar!

---

Todo lo demás OK!